src/main/java/strategy/FruitStorageHandler.java

```diff
@@ -0,0 +1,37 @@
+package strategy;
+
+import db.FruitStorage;
+import java.util.Map;
+
+public class FruitStorageHandler implements StorageHandler<String,Integer> {
```

Comment:
By current design this class is much more connected to DB that to Strategy so it should be placed and named properly
```suggestion
package db;

import java.util.Map;

public class FruitStorageDao implements StorageHandler<String,Integer> {
```